wdfn-server/waterdata/views.py

```diff
@@ -1,10 +1,11 @@
 """
 Main application views.
 """
-import json
-import datetime
+import datetime, json, smtplib
```

Comment:
Codacy is pinging this. When importing an entire module, they should go on separate lines.